### DISCUSSION TITLE
Moutain Duck - Mismatch between MD5 hash notifications on macOS Ventura

### DISCUSSION THREAD

**Discussion by mauricio-it** [iterate-ch/cyberduck#14685] [Q&A]
Hello MoutainDuck Team,

I hope you are keeping well.
We are evaluating your app and hopefully replacing the Google Drive desktop app soon. (as we are facing so many issues with files not syncing). We are happy with the overall performance of Moutain Duck but this alert keeps prompting for a few users. What is it about? How do we fix this?

Thanks a lot.

![Pasted_Image_17_05_2023__11_32](https://github.com/iterate-ch/cyberduck/assets/133867353/9d979f67-fd25-459e-a69a-45e06ab9c710)

**Comment by dkocher** [comment]
The notification should only display in the error case when checksum of the data downloaded does not match what Google Drive advertised in the file metadata. Assuming there is a bug somewhere causing the mismatch in the verification, you can disable the check using the [hidden configuration option](https://docs.mountainduck.io/mountainduck/preferences/#hidden-configuration-options)

```
queue.download.checksum.calculate=false
```
